README.md

```diff
@@ -37,3 +37,8 @@ npm test
 npx eslint . --ext .js,.jsx,.ts,.tsx
```

Comment:
I believe we should remove this line. This is bad practice, because if we come to change the way we run eslint, this may be forgotten. Better stick to npm run eslint, that we can then configure in package.json